accounting/on_chain.go

```diff
@@ -269,13 +269,24 @@ func onChainReport(info onChainInformation) (
 			continue
 		}
 
-		// Finally, if the transaction is unrelated to channel opens or
-		// closes, we create a generic on chain entry for it. We check
-		// our list of known sweeps for this tx so that we can separate
-		// it our from regular chain sends.
-		isSweep := info.sweeps[txn.TxHash]
+		// Next, we check whether our transaction is a sweep. In this
+		// case we also expect to have zero fees, because sweeps use
```

Comment:
Why will it have zero fees?

Reply:
When we use some of the inputs for fees, the wallet doesn't pick up on it in come cases (eg, remote party commit hltcs). Going to fix this in LND so we don't need to do the lookups, this is a workaround. Not always zero though, so updated comment.